src/Components/Components/src/Rendering/Renderer.cs

```diff
@@ -93,7 +93,7 @@ protected internal int AssignRootComponentId(IComponent component)
         /// </summary>
         /// <param name="componentId">The id for the component.</param>
         /// <returns>The <see cref="RenderTreeBuilder"/> representing the current render tree.</returns>
-        private protected ArrayRange<RenderTreeFrame> GetCurrentRenderTreeFrames(int componentId) => GetRequiredComponentState(componentId).CurrentRenderTree.GetFrames();
+        protected ArrayRange<RenderTreeFrame> GetCurrentRenderTreeFrames(int componentId) => GetRequiredComponentState(componentId).CurrentRenderTree.GetFrames();
```

Comment:
Is the expectation that if you get the current frames you can or can't hold on to them? Does this make a copy?

Reply:
It doesn't make a copy. The expectation is that you only access them synchronously.

Reply:
We could rename to “ViewCurrentRenderTreeFrames” to clarify this. Longer term we could use the same technique we plan for ParameterView to throw if you access after the underlying store is mutated further.

Reply:
> ViewCurrentRenderTreeFrames

I'm not actually going to rename it to that, because it's a strange name. The existing name `GetCurrentRenderTreeFrames` is more conventional. We can add the "don't read after mutation" check in the future if we want.